projects/plugins/jetpack/modules/search/customberg/components/header/index.jsx

```diff
@@ -36,7 +35,14 @@ function Header() {
 			</div>
 			<div className="jp-search-configure-header__actions">
 				<SaveButton />
-				<PinnedItems.Slot scope={ COMPLEMENTARY_AREA_SCOPE } />
+				<Button
+					ariaLabel={ __( 'Show settings', 'jetpack' ) }
```

Comment:
should `ariaLabel` here be `aria-label`?

Reply:
Nice catch!